spec/inputs/tcp_spec.rb

```diff
@@ -23,13 +24,14 @@
       plugin = LogStash::Inputs::Tcp.new("codec" => LogStash::Codecs::JSON.new, "port" => 0)
       plugin.register
       insist { plugin.codec }.is_a?(LogStash::Codecs::JSONLines)
+      plugin.teardown
     end
   end
 
-  describe "read plain with unicode", :socket => true do
```

Comment:
With the describe clean, you got rid of the socket filter, is this something we want to do? notice, I'm happy to get rid of most of this filters :-)